runtime/account_test.go

```diff
@@ -192,6 +192,8 @@ func TestRuntimeAccountKeyConstructor(t *testing.T) {
 		},
 	)
 	require.Error(t, err)
+	_ = err.Error()
```

Comment:
Is this to catch panics when calling `.Error()`? I didn't realize that `_` binds an actual name instead of being a reserved wildcard

Reply:
Correct, this checks if the error message can be produced properly. We had cases in the past where the error message function was implemented incorrectly, or the error value itself was prepared incorrectly (e.g. a field was not filled, leading to a nil-dereference in the `Error()` function).